script/Ciel/Day 14/14_01B_01_CIEL14B_KRAKE.txt

```diff
@@ -352,117 +378,128 @@ Its body is completely annihilated by Ciel's black keys.
 -- とどめとばかりに、無防備になった大胸骨に50トンもの<正拳|インパクト>がたたき込まれた。
-And to top it all off, a fifty-ton impact is delivered to its massive sternum that was unprotected.
+Then, as if to finish it off,
+ she delivers a fifty-ton impact to its gargantuan, now-unprotected, sternum.
 }
 [sha:7c50aab13014cb6a8b4cc18ffa1abdc88b506df6]{
 -- Page 33, Offset 38419.
 -- その外見通り、不死身と思われた怪物もここまで破壊されては退場する他ない。
-As one could see through its appearance, the monster that was considered immortal had no choice but to leave after such devastation.
+Just looking at the extent of the destruction,
+ it's clear that even monsters thought to be immortal would be forced to leave the stage after suffering such devastation.
 }
 [sha:6798ceb12b6d16086592e89b01f6c0f40932e952]{
 -- Page 33, Offset 38420.
 -- 真の不死―――限度のない復元者であるシエルと<耐久力|タフネス>で競った時点で、怪物は敗北していたと言える。
-True immortality―――One could argue that the monster had been defeated the moment it competed in toughness against Ciel, who was a limitless regenerative being.
+One could say that this monster had already lost the moment it tried to compete on toughness against Ciel――
+someone possessed of a truly undying body, and whose restoration knows no bounds.
 }
 [sha:cd3bb53cc0fdc446f7b501658704b08c007a3369]{
 -- Page 34, Offset 38421.
 -- 「ハァ……ハァ……ハァーーーー…………」
-"Hah... hah... hah......"
+"%{i}Hah%{/i}... %{i}hah%{/i}... %{i}haaah%{/i}..."
 }
 [sha:7e2881278b502f05da62140259dfcd818c6ff0bc]{
 -- Page 34, Offset 38422.
 -- とはいえ、この戦闘は彼女にとっても予想外のダメージを与えていた。
-Nevertheless, the fight inflicted far more damage to her than she had anticipated.
+Even so, the fight inflicted far more damage to her than she had anticipated.
 }
 [sha:c1f3107894233f5535c97f31144cc2060f3b9552]{
 -- Page 34, Offset 38423.
 -- 体力、肉体の損傷ではない。
-The damage was not on her body or her strength,
+Not against her body, or her stamina―――
 }
 [sha:3029e60c2760369931fa9b9fc53f79f0dbeae1c8]{
 -- Page 34, Offset 38424.
 -- 精神的な傷を、である。
-but rather on her mind.
+But rather against her mind.
 }
 [sha:c2a81b1888277e8275c81a3e538fb74e31001d9a]{
 -- Page 35, Offset 38425.
 -- 「今のは……いったい……？」
-"What the hell was that just now...?"
+"What on earth... was that?"
 }
 [sha:31bb8c7bb0bb928c7a2b832007d7a744a4ced8e2]{
 -- Page 35, Offset 38426.
 -- アレは決して自然発生したものではない。
-It was not a naturally occuring thing by any means.
+It was something that could never have arisen naturally.
 }
 [sha:54f9d954e583f2a9bff5ecc6c26839b3c1c8ef66]{
 -- Page 35, Offset 38427.
 -- 死徒に血を送られて変貌する元人間もいるにはいるが、あそこまでの異形に成ることはない。
-There are cases where humans were transformed after being given blood from Dead Apostles, but never as something grotesque as this.
+There have been cases of humans being transformed by the blood injected into them by Dead Apostles,
+ but never into anything so grotesque as this.
 }
 [sha:109899d6a03464d26f7ebd4d86d9b873911ca700]{
 -- Page 35, Offset 38428.
 -- 考えられるとしたらロアの置き土産だ。
-If she had to guess, it would have to be a parting gift from Roa.
+If she had to guess, it might be a parting gift from Roa.
```

Comment:
Tense
Since it's metaphorical 置き土産, I don't think 'parting gift' is ideal. 
>The only possibility she can conjure is that it had been something left behind by Roa.